__tests__/helpers/pairs.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { parsePairs } from "../../src/helpers/pairs";
```

Comment:
We have support for tests in TS as well now 🎉 I'm thrilled with how nice the interop b/t Reason and TS has been 😄